Packs/Hackuity/Integrations/Hackuity/command_examples.txt

```diff
@@ -0,0 +1,3 @@
+!hackuity-search-findings asset_name=example.com
+!hackuity-search-provider-vulnerabilities asset_name=example.com
+!hackuity-search-vulndb-vulnerabilities asset_name=example.com
```

Comment:
command examples used in building readme, should be example that would return reasonable and correct results, which i assume that example.com does not really work in that case

Reply:
Done

Reply:
where do you mean its done? the file is not updated